test/cases.yml

```diff
@@ -25,10 +25,10 @@
   steps:
     -   in: bn create node8 naughtyvegetable
     -   in: bn deploy naughtyvegetable
-        err: Could not connect to Binaris backend
+        err: "getaddrinfo ENOTFOUND fake.binaris.neverworks fake.binaris.neverworks:443"
         exit: 1
     -   in: bn remove naughtyvegetable
-        err: Could not connect to Binaris backend
+        err: "Error: getaddrinfo ENOTFOUND fake.binaris.neverworks fake.binaris.neverworks:443"
```

Comment:
why are the error messages different?
(I don't mean technically why, I mean specification wise. They should be similar)

Reply:
If you're talking about between `deploy` and `remove` it's because `deploy` uses `request` whereas `remove` uses `request-promise-native`. I could do some specific coercion  on the `deploy` to make it fit with the traditional error but wasn't sure how important it is.